### PR TITLE
Add tensorflow 1.x integration to track weights & biases distributions via aim SDK.

### PR DESCRIPTION
**Pull request by bhaveshk658**
From the previous pull request, a check for whether the module was created in TensorFlow was added. In addition, the order of the names of the layers is no longer sorted alphabetically when viewing through the board to maintain the ordering of the NN.

### CODE REVIEW

File: aim/engine/utils.py [hunk 1]
```diff
@@ -121,7 +121,12 @@ def is_pytorch_module(inst):
     """
     return inst_has_typename(inst, ['torch', 'Module'])
 
-
+def is_tensorflow_session(inst):
+    """
+    Check whether `inst` is isntance of tensorflow session
+    """
+    return inst_has_typename(inst, ['session'])
```
Comment (gorarakelyan) [inline aim/engine/utils.py:128]: Is only checking session enough? Maybe we should also check if an instance has `tf`, `tensorflow` in its class type

**Comment by gorarakelyan** [comment]
@bhaveshk658 Have you also tested for `neural_net_with_ops` model?

**Comment by bhaveshk658** [comment]
I think it looks alright with that model as well:
Screen Shot 2020-06-12 at 9 40 43 AM

File: aim/engine/utils.py (hunk 1 above)
Comment (bhaveshk658) [reply aim/engine/utils.py:128]: Added 'tensorflow' to types.

**Comment by gorarakelyan** [comment]
Good job!

**Pull request closed without merging**